tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/temporal/utils.scala

```diff
@@ -35,6 +37,22 @@ case class PredExs(
 
 package object utils {
   val builder: ScopedBuilder = new ScopedBuilder()
```

Comment:
There's a convenience-like package for the builder, you can just import that instead of defining your own.

Reply:
Where is it? I did a search for ScopedBuilder that didn't turn up anything

Reply:
Ah, my bad, this was part of the experimental PR that got scrapped.